registry/storage/linkedblobstore.go

```diff
@@ -241,6 +250,132 @@ func (lbs *linkedBlobStore) linkBlob(ctx context.Context, canonical distribution
 	return nil
 }
 
+func (lbs *linkedBlobStore) Enumerate(ctx context.Context, digests []digest.Digest, last string) (n int, err error) {
+	context.GetLogger(ctx).Debug("(*linkedBlobStore).Enumerate")
+	bufs := make([][]digest.Digest, len(lbs.blobsRootPathFns))
```

Comment:
Would it be easier to make this a `[]digest.Digest` and append the `found` array?